crates/jstz_cli/src/sandbox/mod.rs

```diff
@@ -10,20 +10,19 @@ mod daemon;
 use crate::config::Config;
 
 #[derive(Subcommand)]
-pub enum Command {
+pub enum SandboxCommand {
     /// Starts a sandbox.
-    Start,
+    Start {
+        /// Do not daemonize the process.
+        #[clap(long, short, default_value = "false")]
+        no_daemon: bool,
+    },
```

Comment:
The standard flag for this is `--detach` (or `-d` for short). By default it is `false`, when set it detaches (daemonizes) the process

Reply:
The point of the flag is to NOT daemonize  the process. I don't think there is any standard for it though a possible alternative is --attach. However, since there is no point in using it I have decided to hide it as it is only used internally.

Reply:
There is a standard, namely _all_ long-lived processes don't automatically detach unless explicitly specified. 

`--detach` is inspired by `docker run --detach`

For all examples of not implicitly detaching long-lived processes, simply look at:
- `wrangler dev`
- `vercel dev`
- Most HTTP dev servers for frontends (e.g. react, vue, etc)

Reply:
changed it so that it does both --detach and without --detach (required some changes to how stopping/logging is handled). However thinking about it, I don't like this change  much - using it detached seems to me much better and easier to use to me and should be a default option IMO. I would rather have --attach flag and for sandbox start without any flag I would just output a line saying that it was started in background and can be killed with sandbox stop.

Reply:
I think we'll simply have to agree to disagree here